armi/reactor/blueprints/isotopicOptions.py

```diff
@@ -394,6 +394,7 @@ def apply(self, material):
         material.massFrac = dict(self.massFracs)
         if self.density is not None:
             if not isinstance(material, materials.Custom):
+                # TODO I think this warning should go away?
```

Comment:
I would like a second set of eyes on the runLog statement below. I am 99% sure that that print should be removed completely, because we are now allowing for non-`Custom` materials to have density specified.

Reply:
@bsculac

Reply:
The runlog appears to still be correct and applicable, as the density is not applied to `material` in this function if  `material` is not a custom material per line 396. I don't recognize anywhere in the changes that the density of a library material is being set, am I missing it?

Related: I believe this was intended to explicitly call out that setting a custom density for a library material using custom isotopics is not allowed. Presumably a library material would have known, vetted properties that should not be overwritten by users in a passive way. From recollection, using custom isotopics on a library material allows the density to be set on a component specifically using the custom isotopics, but using custom isotopics to set the density for the reference material is no bueno.

Reply:
I think you're right. 

The thing that keeps tripping me up is the last sentence: "Only custom materials may have a density specified."

I feel like that can be reworded to make the intention more clear. Right now I read that and think that its saying "the density that you specified is doing nothing".

I'll try to rejigger a bit.

Reply:
Yeah, I vaguely remember struggling how to phrase this without being very wordy. Some ideas: 

"Only custom materials may have a density specified using custom isotopics, though components using custom materials may have their densities changed." 

"The reference density of materials in the materials library will not be changed by custom isotopics, but components with custom isotopics using a library material will use the density implied by the custom isotopics."

Reply:
Alright, I updated the wording. Thanks for helping here.